ingestion/nse-scraper/src/main.py

```
@@ -0,0 +1,167 @@
+        elif scraper_type == "trading-calendar":
+            from src.scrapers.trading_calendar import TradingCalendarScraper
+            scraper = TradingCalendarScraper()
+            scraper.scrape(year=target_date.year, dry_run=dry_run)
+        else:
+            console.print(f"[red]Unknown scraper type: {scraper_type}[/red]")
+            raise typer.Exit(1)
+
+        console.print(f"[green]✓[/green] Successfully scraped {scraper_type}")
+
+    except Exception as e:
+        logger.error(f"Scraper failed", scraper=scraper_type, error=str(e), exc_info=True)
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)